src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -7566,6 +7565,12 @@ public override BoundNode VisitDeconstructValuePlaceholder(BoundDeconstructValue
             return null;
         }
 
+        public override BoundNode VisitLValuePlaceholder(BoundLValuePlaceholder node)
+        {
+            SetUnknownResultNullability(node);
```

Comment:
>SetUnknownResultNullability(node); [](start = 12, length = 34)

I guess unknown would work, but, given the usage, should we use SetNotNullResult instead?

Reply:
This is what we do in ```VisitImplicitReceiver``` 

---
In reply to: [325386703](https://github.com/dotnet/roslyn/pull/38732#discussion_r325386703) [](ancestors = 325386703)